.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature
+about: Outline a proposed feature
+
+---
+
+## TL;DR
```

Comment:
Would I be being too old fashioned if I wanted to call this a summary?

Reply:
https://github.com/GafferHQ/gaffer/commit/765e7b32d8bce2e3ca5da10c69390869db02ad67